market/connector/docker-connector/meta.json

```diff
@@ -0,0 +1,22 @@
+{
+    "$schema": "https://json-schema.axonivy.com/market/10.0.0/meta.json",
+    "id": "docker-connector",
+    "name": "Docker Connector",
+    "description": "Use the Docker Connector to create, start, stop, and remove Docker containers.",
```

Comment:
I would go for (@linhpd-axonivy): Use this connector to create, start, stop, and remove Docker containers.